package.json

```
@@ -1,39 +1,20 @@
 {
   "name": "hackafor",
-  "type": "module",
-  "version": "0.0.0",
-  "private": true,
+  "version": "1.0.0",
   "scripts": {
-    "dev": "vite",
-    "build": "vite build",
-    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
-    "lint:fix": "npm run lint -- --fix",
-    "preview": "vite preview",
+    "dev": "pnpm --recursive dev",
+    "lint": "pnpm --recursive lint",
+    "fix": "pnpm --recursive fix",
```

Proba que anden por las dudas, que estoy sugeriendo desde GitHub
```suggestion
    "dev": "pnpm run -r dev",
    "lint": "pnpm run -r lint",
    "lint:fix": "pnpm run -r lint:fix",
```